ecosystem/sep-0024.md

```diff
@@ -73,6 +74,54 @@ To support the exchange of non-equivalent assets using this protocol, the anchor
+
+#### Deposit:
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/deposit/interactive`](#deposit) using the parameter `asset_code`.
+1. The user proceeds with the off-chain deposit, sending `BRL` to the Anchor's bank account.
+1. Upon receiving the off-chain deposit funds, the Anchor converts it to `USDC` using the market rate and sends the resulting amount to the user's Stellar account.
+
+#### Withdraw:
+
+Considering the user will be making a Stellar payment of `USDC` in exchange for withdrawing `BRL` in the bank account:
+
+1. The Wallet checks the assets' market conversion rate with [`SEP-38 GET /price`](sep-0038.md#get-price) and shows the estimated price to the user.
+1. The Wallet initiates a [`POST /transactions/withdraw/interactive`](#withdraw) using the parameter `asset_code`.
+1. The user proceeds with the on-chain payment, sending `USDC` to the Anchor's Stellar account.
+1. Upon receiving the on-chain withdraw funds, the Anchor converts it to `BRL` using the market rate and sends the resulting amount to the user's bank account.
+
+### Asset Conversions Using a Firm Quote
+
+With firm quotes, users are guaranteed an asset price they will pay if they submit the payment before the quote expiration. Here is how the conversion should happen in deposits and withdrawals for an asset pair like `USDC <> BRL`:
+
+#### Deposit:
+
+Considering the user will be making a bank deposit of fiat `BRL` in exchange for `USDC` in the Stellar network:
+
+1. The Wallet gets a firm conversion rate in [`SEP-38 POST /quote`](sep-0038.md#post-quote) and shows the price and expiration date to the user.
+1. The Wallet initiates a [`POST /transactions/deposit/interactive`](#deposit) using the parameter `quote_id` returned from the previous step, as well as `asset_code`.
+1. The user proceeds with the off-chain deposit, sending `BRL` to the Anchor's bank account before the quote expires.
+1. Upon receiving the off-chain deposit funds, the Anchor converts it to `USDC` according with the agreed quote and sends the resulting amount to the user's Stellar account.
+1. If the user sends the funds after the quote expiration, the Anchor can proceed according with its own policy.
+
+#### Withdraw:
+
+Considering the user will be making a Stellar payment of `USDC` in exchange for withdrawing `BRL` in her bank account:
+
+1. The Wallet gets a firm conversion rate in [`SEP-38 POST /quote`](sep-0038.md#post-quote) and shows the price and expiration date to the user.
+1. The Wallet initiates a [`GET /withdraw-exchange`](#withdraw-exchange) using the parameter `quote_id` returned from the previous step, as well as `asset_code` and `destination_asset`.
```

Comment:
There is no `destination_asset` parameter in SEP-24's endpoint, and no `-exchange` endpoint.